free-2.md

```diff
@@ -58,12 +68,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I would need to spend time determining what language I wanted to implement my DSL in and how I might make use of existing code basis and programs since many things have been made which are similar to this, and I would just be combining them in a new way.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+Depending on what existing work I leverage this could be either a very big project or a very reasonable project but I might spend a lot of time determining how to leverage and intermingle existing programs.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This seems like a very interesting domain to work in and one that would be very useful, but there is already a lot of work that has been done in this area so its not very original. I would need to think more about what unique aspects my program would bring to its users.
+
```

Comment:
This is a really good point--I think a project is more satisfying when you think of some way to target a new need or use a different approach to solve an old problem. When I'm trying to brainstorm how to approach a problem uniquely or address an under-served need, I usually try to think about it from the point of view of someone else I know. Trying to consider how someone else might solve the problem sometimes opens up new ideas.